helm_deploy/values-prod.yaml

```diff
@@ -12,6 +12,7 @@ generic-service:
     HMPPS_AUTH_URL: https://sign-in.hmpps.service.justice.gov.uk/auth
     PRISON_API_URL: https://api.prison.service.justice.gov.uk
     APPLICATIONINSIGHTS_CONFIGURATION_FILE: applicationinsights.json
+    FEATURE_EVENTS_SNS_ENABLED: false
```

Comment:
Not sure what is required here @philip-milne and @rj-adams  what is the missing step here, surly I have to link this up in the cloud some how?

Reply:
If the values-prod.yaml defines the default value for the SNS switch in the prod environment, why is there not a similar line in the values.yaml which defines the default value for the SNS switch for other environments?

Or is there another mechanism which does this? and if so, why do we use different mechanisms for different environments?

Reply:
it defaults to true...

Reply:
how does it default to true?

Reply:
@philip-milne informed me of the convention as follows :

feature:
  events:
    sns:
      enabled: true

is converted into the Eq of the below env variable

    FEATURE_EVENTS_SNS_ENABLED

which is then picked up by 

  @Value("\${feature.events.sns.enabled::true}")
  private val snsEventsEnabled: Boolean